cloud/amazon/iam.py

```diff
@@ -663,7 +711,7 @@ def main():
 
         elif state == 'update' and not user_exists:
             module.fail_json(
-                msg="The user %s does not exist. No update made." % name)
+                msg="The user %s does not exit. No update made." % name)
```

Comment:
That is just a typo I corrected...


On 17 March 2016 at 16:18, Michael Scherer <notifications@github.com> wrote:

> In cloud/amazon/iam.py
> <https://github.com/ansible/ansible-modules-core/pull/3264#discussion_r56570638>
> :
>
> > @@ -663,7 +711,7 @@ def main():
> >
> >          elif state == 'update' and not user_exists:
> >              module.fail_json(
> > -                msg="The user %s does not exist. No update made." % name)
> > +                msg="The user %s does not exit. No update made." % name)
>
> I suspect this is not supposed to be part of the PR ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ansible/ansible-modules-core/pull/3264/files/d1b8affd96e07afc984923c78a04c57329963b70#r56570638>
>



-- 
Pierre Jodouin
Cloud Solutions Architect